docs/src/main/asciidoc/security-authentication-mechanisms.adoc

```diff
@@ -35,7 +35,10 @@ The following table maps specific authentication requirements to a supported mec
 
 |Username and password |xref:security-basic-authentication.adoc[Basic], <<form-auth>>
 
-|Bearer access token |xref:security-oidc-bearer-token-authentication.adoc[OIDC Bearer token authentication], xref:security-jwt.adoc[JWT], xref:security-oauth2.adoc[OAuth2]
+|Bearer access token |xref:security-oidc-bearer-token-authentication.adoc[OIDC Bearer token authentication], xref:security-jwt.adoc[JWT]
+ifndef::no-quarkus-elytron-security-oauth2[]
+, xref:security-oauth2.adoc[OAuth2]
```

Comment:
@rolfedh Will it end up on a separate line ?

Reply:
Good question. No it renders on the same line.

Reply:
Shown here with line wrap between the comma and `OAuth2`.
![image](https://github.com/quarkusio/quarkus/assets/350907/90b25ccc-039e-482a-819d-b6f275df23c8)